lib/menu.ts

```diff
@@ -220,7 +239,7 @@ export function menuItemSubmenu(
 
   const popupOptions: IMenuOptions = {
     placement: 'right-start',
-    trigger: ['click'],
+    trigger: [],    // no "click": don't toggle this menu on click.
```

Comment:
Toggling submenus on click seemed wrong in practice. The menu is already opened on mouseover. If the user moves the mouse and clicks immediately, it will flicker open and close right away. Just letting it open on mouseover feels better.